pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux_test.go

```diff
@@ -682,17 +700,20 @@ func Test_managerImpl_processShutdownEvent(t *testing.T) {
 				clock:          fakeclock,
 				dbusCon:        &fakeDbus{},
 			},
-			wantErr:                false,
-			expectedOutputContains: "Shutdown manager pod killing time out",
+			wantErr:              false,
+			exceptOutputContains: "Shutdown manager pod killing time out",
 		},
 	}
-
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			logger, _ := ktesting.NewTestContext(t)
+			l := klog.Level(1)
+			l.Set("1")
+			// hijack the klog output
+			tmpWriteBuffer := new(buffer)
+			klog.SetOutput(tmpWriteBuffer)
+			klog.LogToStderr(false)
```

Comment:
Should we defer to revert this with `LogToStderr(true)`?

Reply:
Same here - I am just reverting.